### ISSUE TITLE
Stage amp volume slider possible?

### ISSUE THREAD

**Issue by TrackDownPromotion** [Guribo/UdonVoiceUtils#11]
Hello!
Ive set up stageamp for a club world but the issue is not everyone wanna hear ausio from the stage at full blast all the time.
Is there a way to attach the boost amp to a local volumeslider for each individual to use in the world and individually set the volume of ones liking?

**Comment by Guribo** [comment]
Yes, you can create a small script that is controlled by a slider. Whenever the slider is moved the script can change the max. Voice range on the Audio Override of the stage. This will only affect the local player.

**Comment by Flare26** [comment]
Can confirm, I use a script that modifies the gain of a public amp zone for people further away without upping the gain for people close to the amp zone. Sadly you cannot yet fully escape the low pass filter VRChat begins to apply as volume falls off

**Comment by Guribo** [comment]
Untick or set this box to false in case you use my systems, otherwise please check the documentation of the VRCPlayerApi for this option.

VRC broke this in the past to my knowledge and I am not sure if it currently is fixed or not to be honest.

![image](https://github.com/Guribo/UdonVoiceUtils/assets/49484893/f97e9ef5-bb58-4a79-bcd0-4fe6ed640087)

**Comment by Guribo** [comment]
If this is still an issue please upvote my canny: https://feedback.vrchat.com/bug-reports/p/vrcplayerapisetvoicelowpassfalse-no-longer-has-any-effect
